[2025-03-27T12:03:57.312+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: insert_aqi_to_postgres.fetch_and_insert_aqi scheduled__2025-03-27T11:00:00+00:00 [queued]>
[2025-03-27T12:03:57.316+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: insert_aqi_to_postgres.fetch_and_insert_aqi scheduled__2025-03-27T11:00:00+00:00 [queued]>
[2025-03-27T12:03:57.316+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-03-27T12:03:57.323+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): fetch_and_insert_aqi> on 2025-03-27 11:00:00+00:00
[2025-03-27T12:03:57.325+0000] {standard_task_runner.py:60} INFO - Started process 1979 to run task
[2025-03-27T12:03:57.327+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'insert_aqi_to_postgres', 'fetch_and_insert_aqi', 'scheduled__2025-03-27T11:00:00+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/insert_aqi_to_db.py', '--cfg-path', '/tmp/tmpma004ysq']
[2025-03-27T12:03:57.328+0000] {standard_task_runner.py:88} INFO - Job 34: Subtask fetch_and_insert_aqi
[2025-03-27T12:03:57.333+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-27T12:03:57.347+0000] {task_command.py:423} INFO - Running <TaskInstance: insert_aqi_to_postgres.fetch_and_insert_aqi scheduled__2025-03-27T11:00:00+00:00 [running]> on host 79a5714298e2
[2025-03-27T12:03:57.376+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='insert_aqi_to_postgres' AIRFLOW_CTX_TASK_ID='fetch_and_insert_aqi' AIRFLOW_CTX_EXECUTION_DATE='2025-03-27T11:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-27T11:00:00+00:00'
[2025-03-27T12:03:57.632+0000] {insert_aqi_to_db.py:37} INFO - âœ… AQI in Bangkok: 110 at 2025-03-27 12:03:57.632236
[2025-03-27T12:03:57.643+0000] {base.py:83} INFO - Using connection ID 'postgres_conn' for task execution.
[2025-03-27T12:03:57.647+0000] {sql.py:450} INFO - Running statement: 
                INSERT INTO air_quality_data (city, aqi, timestamp_utc)
                VALUES (%s, %s, %s);
            , parameters: ('Bangkok', 110, datetime.datetime(2025, 3, 27, 12, 3, 57, 632236))
[2025-03-27T12:03:57.648+0000] {sql.py:459} INFO - Rows affected: 1
[2025-03-27T12:03:57.649+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-03-27T12:03:57.655+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=insert_aqi_to_postgres, task_id=fetch_and_insert_aqi, execution_date=20250327T110000, start_date=20250327T120357, end_date=20250327T120357
[2025-03-27T12:03:57.686+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-27T12:03:57.695+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
